httpapi/httpapi_test.go

```diff
@@ -7,33 +7,12 @@ import (
 	"net/http"
 	"testing"
 
-	"github.com/ipfs/go-datastore"
 	"github.com/libp2p/go-libp2p-core/peer"
-	"github.com/libp2p/hydra-booster/node"
-	"github.com/libp2p/hydra-booster/opts"
-	"github.com/multiformats/go-multiaddr"
+	hytesting "github.com/libp2p/hydra-booster/testing"
```

Comment:
Why `testing` and not `test?

Reply:
Because the golang module is called "testing" and I read a recommendation on a blog to call it "testing".

Reply:
Good to know! thanks